From 91e39f4ad210e41f36167f3927b15e810b6edc15 Mon Sep 17 00:00:00 2001
In-Reply-To: <20201201094623.4290-1-avarab@gmail.com>
References: <20201201094623.4290-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 1 Dec 2020 10:47:55 +0100
Subject: [PATCH] MaintNotes: use https:// instead of git:// when possible
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change advertised git:// links to https://. These all work as
arguments to "git clone", but in addition they also have friendly web
interfaces.

This leaves just git://ozlabs.org/~paulus/gitk as the only git://
URL. As far as I can tell there's no web interface for it. There is
e.g. https://git.ozlabs.org/?p=ppp.git which is a frontend for
git://git.ozlabs.org/~paulus/ppp.git, but even though cloning the repo
at git://git.ozlabs.org/~paulus/gitk.git works (not the "git"
subdomain) there's no corresponding web UI at
https://git.ozlabs.org/?p=gitk.git

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
Not on top of the last patch I sent (in In-Reply-To), but it
auto-merges.

 MaintNotes | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/MaintNotes b/MaintNotes
index 0dc03080de..b52b164e2d 100644
--- a/MaintNotes
+++ b/MaintNotes
@@ -132,15 +132,15 @@ leaking out before we're ready to make public announcements.
 
 My public git.git repositories are (mirrored) at:
 
-  git://git.kernel.org/pub/scm/git/git.git/
+  https://git.kernel.org/pub/scm/git/git.git/
   https://kernel.googlesource.com/pub/scm/git/git
-  git://repo.or.cz/alt-git.git/
+  https://repo.or.cz/alt-git.git/
   https://github.com/git/git/
 
 This one shows not just the main integration branches, but also
 individual topics broken out:
 
-  git://github.com/gitster/git/
+  https://github.com/gitster/git/
 
 A few web interfaces are found at:
 
@@ -151,8 +151,8 @@ A few web interfaces are found at:
 Preformatted documentation from the tip of the "master" branch can be
 found in:
 
-  git://git.kernel.org/pub/scm/git/git-{htmldocs,manpages}.git/
-  git://repo.or.cz/git-{htmldocs,manpages}.git/
+  https://git.kernel.org/pub/scm/git/git-{htmldocs,manpages}.git/
+  https://repo.or.cz/git-{htmldocs,manpages}.git/
   https://github.com/gitster/git-{htmldocs,manpages}.git/
 
 The manual pages formatted in HTML for the tip of 'master' can be
-- 
2.29.2.222.g5d2a92d10f8

